repos/system_upgrade/el7toel8/actors/checkosrelease/libraries/library.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+from leapp.exceptions import StopActorExecution, StopActorExecutionError
+from leapp.libraries.common import reporting
+from leapp.libraries.stdlib import api
+from leapp.models import OSReleaseFacts
+
+
+def skip_check():
+    """ Check if has environment variable to skip this actor checks """
+    if os.getenv('LEAPP_SKIP_CHECK_OS_RELEASE'):
+        reporting.report_generic(
+            title='Skipped OS release check',
+            severity='low',
+            summary='OS release check skipped via LEAPP_SKIP_CHECK_OS_RELEASE env var.',
+        )
+        raise StopActorExecution()
+
+
+def check_os_version(supported_version):
+    """ Check OS version and inhibit upgrade if not the same as supported ones """
+    if not supported_version:
+        raise StopActorExecution()
```

Comment:
That is an usage error to me, not a reason without comment to end the execution - This should blow up in flames ;D

Reply:
Lol, ok I will change :)

Reply:
Well this still doesn't go up in flames. But well this just shouldn't happen and I am not even sure if it is necessary to check for this at all.